### DISCUSSION TITLE
Ending support for Rhino v5

### DISCUSSION THREAD

**Discussion by michaelhoehn** [BHoM/BHoM#1437] [General]
Since BHoM currently supports Rhino v5 it's become visible that we need to update our Rhino Common version support to be able to implement the latest features in Rhino v6 (potentially another conversation should be had for v7 support if/when that's needed). 
 
As seen in a recent PR developed to support Rapid Prototyping workflows, Rhino 6 common has been updated in a way that is no longer supporting certain Rhino 5 features. In this [PR](https://github.com/BHoM/Rhinoceros_Toolkit/pull/230), screen capture settings have been updated to support a broader feature set than was previously offered in v5. But in order to implement this new PR on main, we would effectively stop supporting Rhino v5 within the BHoM. This may be a small example, but I'm certain there will be others. 
 
Please add your thoughts, for or against this implementation so we can discuss and move on. Ideally, I'd like to close this out this sprint in favour of no longer supporting v5 so I can integrate new features. Happy to consider any and all opinions. I'm sure there may be considerations from other disciplines, but need to get them on the table first! 

FYI @FraserGreenroyd @al-fisher @IsakNaslundBh @pawelbaran @enarhi

**Comment by FraserGreenroyd** [comment]
Hi @michaelhoehn my tuppence for what it's worth - I would agree that it's time to stop supporting Rhino 5 as McNeel are promoting Rhino 6, 7, and Rhino 8 is in WIP I believe.

My only concern would be ensuring we've fully tested Rhino_Toolkit and Grasshopper_UI in the versions of Rhino we wish to support, as we have [previously had issues](https://github.com/BHoM/Grasshopper_UI/pull/643) with supporting higher versions with the current code base. So providing we're happy to do a full test and review against the versions we choose to support, then I would have no qualms personally with us jumping up to Rhino 6 and/or Rhino 7 and ditching support for Rhino 5 from the 6.0 beta release at the end of this year 😄 

Happy to see others thoughts though, that's just my personal viewpoint.
